packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -198,8 +198,8 @@ const DefaultSummaryConfiguration: ISummaryConfiguration = {
 
     maxTime: IdleDetectionTime * 12,
 
-    // Snapshot if 1000 ops received since last snapshot.
-    maxOps: 1000,
+    // Snapshot if 100 ops received since last snapshot.
+    maxOps: 100,
```

Comment:
Let's separate work into multiple smaller PRs. This change will depend on https://github.com/microsoft/FluidFramework/pull/9965 going through first (stop using policy from service), so at the moment this is noop and we should not add it here (for now)